include/sound/sof/topology.h

```diff
@@ -57,8 +57,8 @@ struct sof_ipc_comp {
 	uint32_t pipeline_id;
 	uint32_t core;
 
-	/* reserved for future use */
-	uint32_t reserved[1];
+	/* extended data offset, 0 if no extended data */
+	uint32_t ext_data_offset;
```

Comment:
There are now two ways to define semantics of this field -> This now needs to be aligned with https://github.com/thesofproject/sof/pull/2920/commits/31e8566bd4d9680e0ad8b89c7bcaf40997ca6c8a#r450945347

Reply:
Thanks @kv2019i , yes, we need to wait until that is aligned.